cors/src/main/java/io/helidon/cors/CorsSupportHelper.java

```diff
@@ -369,14 +370,68 @@ public Aggregator aggregator() {
         return aggregator;
     }
 
-    private boolean isRequestTypeNormal(CorsRequestAdapter<Q> requestAdapter, boolean silent) {
+    // For testing
+    static RequestTypeInfo isRequestTypeNormal(String originHeader, UriInfo requestedHostUri) {
```

Comment:
Sorry; missed this before. Kind of an odd method name given that it does not return a `boolean`.

Reply:
Renamed.